specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -2433,6 +2439,9 @@
           "readOnly": true,
           "$ref": "#/definitions/ReplicationStatus"
         },
+        "validationsProfile": {
```

Comment:
let's put this at end of the list. So after security Profile. 
This is to preserve parameters list order

Reply:
@lkumaar

Reply:
done